system/common.nix

```diff
@@ -0,0 +1,107 @@
+{ lib, pkgs, ... }:
+
+let
+  systems = {
+    darwin = "x86_64-darwin";
+    linux = "x86_64-linux";
+  };
+  linux = import ./linux.nix { inherit lib pkgs; };
+  osx = import ./osx.nix { inherit pkgs; };
+  zshCustom = pkgs.callPackage ../config/modSteeefZsh.nix { };
+
+  packages = [
```

Comment:
`packages = with pkgs; [` then you can remove `pkgs.` from the front of everything within the array